### PR TITLE
Fix Issue #18

### PR DESCRIPTION
**Pull request by fAns1k**
Issue #18

### CODE REVIEW

**Comment by cybergrind** [comment]
I believe we should mention this in readme too.

File: example/android/gradle.properties [hunk 1]
```diff
@@ -18,3 +18,4 @@
 # org.gradle.parallel=true
 
 android.useDeprecatedNdk=true
+org.gradle.daemon=true
```
Comment (cybergrind) [inline example/android/gradle.properties]: This isn't related to fix, better to push this in other commit

Comment (fAns1k) [inline example/android/gradle.properties]: ок

**Comment by itsmepetrov** [comment]
@fAns1k please add info about this issue in `README.md`

**Comment by itsmepetrov** [comment]
Looks good to merge

**Pull request merged**